parity-util-mem/Cargo.toml

```diff
@@ -19,6 +19,7 @@ wee_alloc = { version = "0.4.5", optional = true }
 # The performance penalty is only around 3% on average over our benchmarks.
 mimallocator = { version = "0.1.3", features = ["secure"], optional = true }
 mimalloc-sys = { version = "0.1.6", optional = true }
+parity-util-mem-derive = { path = "derive", version = "0.1" }
```

Comment:
Need to remove usage of [`malloc_size_of_derive`](https://github.com/paritytech/parity-common/pull/291/files#diff-a6e9f06585ce3cce1d6deea24a49aeb1R12) right?